napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -333,22 +538,39 @@ def addItem(
                 self.handle_action, item, pkg_name, InstallerActions.CANCEL
             )
         )
-        item.setSizeHint(widg.sizeHint())
-        self.setItemWidget(item, widg)
+
+        item.setSizeHint(item.widget.size())
+        widg.install_info_button.setDuration(0)
+        widg.install_info_button.toggled.connect(
+            lambda: self._resize_pluginlistitem(item)
+        )
+
+    def _resize_pluginlistitem(self, item):
+        """Resize the plugin list item, especially after toggling QCollapsible."""
+        height = item.widget.height()
+        if item.widget.install_info_button.isExpanded():
+            item.widget.setFixedHeight(int(height * SCALE))
+        else:
+            item.widget.setFixedHeight(int(height / SCALE))
+        item.setSizeHint(item.widget.size())
```

Comment:
It looks problematic. Where do we set the initial height? Why can we not use constant and not play with rounding problems?

Reply:
The height is scaled automatically when the widget is created.   Can we save this one for another PR?

Reply:
ok,  but please update proper issue